doc/source/_static/custom.css

```diff
@@ -39,6 +39,133 @@ html[data-theme="dark"] {
   border: none;
 }
 
+/* -------------------- HOMEPAGE -------------------- */
+article.bd-article section {
+  padding: 3rem 0 7rem;
+}
+
+h1.homepage-title {
+  font-weight: bold;
+  font-size: 48px;
+  text-align: center;
+  margin-bottom: 4rem;
+}
+
+h1.homepage-title .headerlink {
+  visibility: hidden;
+  position: absolute;
+}
```

Comment:
I would keep this, it is hidden by default but it is nice to easily get a link to specific sections to share.

Reply:
I sorta changed the CSS but tried to keep the header links in the h1 tags (I didn't do it for h3 though). I don't think it was possible to grab the headerlinks before when there was no `id` being generated.